src/ralph/scan/plugins/ssh_cisco_asa.py

```diff
@@ -110,7 +110,8 @@ def scan_address(ip_address, **kwargs):
         ssh.close()
     pairs = parse.pairs(lines=[line.strip() for line in lines])
     sn = pairs.get('Serial Number', None)
-    model, ram, cpu = pairs['Hardware'].split(',')
+    # hardware = model, ram, cpu, corenum*
+    hardware = pairs['Hardware'].split(',')
```

Comment:
You can write it as:
```
model, ram, cpu, sorenums = pairs['Hardware'].split(',')
```
then the comment is useless ( + update rest the code to reflect the change).